### ISSUE TITLE
Update return types.

### ISSUE THREAD

**Issue by laurentmuller** [arodygin/datatables-bundle#17]
I have a project that use this bundle with Symfony version 5.4. When I run PHP unit tests. Some warning are output:

`1x: Method "JsonSerializable::jsonSerialize()" might add "mixed" as a native return type declaration in the future. Do the same in  implementation "DataTables\Column" now to avoid errors or add an explicit @return annotation to suppress this message.`

`1x: Method "JsonSerializable::jsonSerialize()" might add "mixed" as a native return type declaration in the future. Do the same in implementation "DataTables\Order" now to avoid errors or add an explicit @return annotation to suppress this message.`

`1x: Method "JsonSerializable::jsonSerialize()" might add "mixed" as a native return type declaration in the future. Do the same in implementation "DataTables\Search" now to avoid errors or add an explicit @return annotation to suppress this message.`

`1x: Method "JsonSerializable::jsonSerialize()" might add "mixed" as a native return type declaration in the future. Do the same in implementation "DataTables\DataTableQuery" now to avoid errors or add an explicit @return annotation to suppress this message.`

To stop all this warnings, just modify code to add a return type. For example for `DataTables\Column` warning:

Before:
```php
   /**
     * {@inheritdoc}
     */
    public function jsonSerialize()
    {
        ...
    }
```
After:
```php
 /**
   * {@inheritdoc}
   * 
   * @return mixed
   */
   public function jsonSerialize()
    {
        ...
    }
```

**Comment by webinarium** [comment]
Hi, thank you for the report.

I have released new version 2.4.1, could you please check and let me know whether it works for you.

**Comment by laurentmuller** [comment]
God job !  it's now working now without warning.

Thank You
